Dockerfile

```diff
@@ -215,7 +204,7 @@ RUN apt-get install -y libgl1-mesa-glx && \
     pip install xvfbwrapper && \
     /tmp/clean-layer.sh
 
-RUN pip install --upgrade mpld3 && \
+RUN pip install mpld3 && \
```

Comment:
No need for the `--upgrade`, that package isn't present anyway and the latest version is downloaded.